dev/PipsPager/PipsPager_themeresources_v2.5.xaml

```diff
@@ -1,188 +0,0 @@
-﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
-<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-
-    <ResourceDictionary.ThemeDictionaries>
-        <ResourceDictionary x:Key="Light">
```

Comment:
Have you verified that the pips pager gets an appropraite style when the app developer uses XamlControlsResources.Version=2dot5?  Since this style is deleted let make sure the other one gets picked up.

Reply:
@StephenLPeters yup, I did, it works:)